src/auto/auto.yml

```diff
@@ -1,52 +1,60 @@
-# Orb Version 2.1.0
+# Orb Version 2.2.0
 
 version: 2.1
 description: Publish NPM packages and canary deployments with Intuit's Auto
-
 orbs:
   yarn: artsy/yarn@5.1.3
   auto: auto/release@0.2.3
   utils: artsy/orb-tools@0.5.0
-
 executors:
   node:
+    parameters:
+      node-version:
+        type: string
+        default: "14.18.1"
     docker:
-      - image: cimg/node:14.18.1
-
+      - image: cimg/node:<< parameters.node-version >>
 jobs:
-  # Publishes a package to NPM
   publish:
-    executor: node
-    environment:
-      AUTO_VERSION: << parameters.version >>
     parameters:
+      node-version:
+        type: string
+        default: "14.18.1"
```

Comment:
**question:** with this we are able to change it in one spot theoretically when upgrading? Is this defined on job level?